src/components/TextInputFocusable/index.android.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {TextInput} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import themeColors from '../../styles/themes/default';
+
+/**
+ * On native layers we like to have the Text Input not focused so the user can read new chats without they keyboard in
+ * the way of the view
+ * In Android I need selection prop for TextInput but for IOS selection prop have some issues
```

Comment:
Suggestion: 

On Android, the selection prop is required on the TextInput but this prop has issues on IOS <LINK_TO_REACTNATIVE_ISSUE>

Copy and paste that same thing to index.ios.js